### DISCUSSION TITLE
speed for openai TTS

### DISCUSSION THREAD

**Discussion by jck411** [KoljaB/RealtimeTTS#212] [General]
I am not confident enough to contribute code yet  but it feels like it would be easy for someone experienced to add "speed" to openai_engine.py ?

the API docs have it as a paramater:

response = client.audio.speech.create(
  model="tts-1",
  voice="alloy",
  input="blah blah",
  speed="2" # 0.25 to 4.0. default is 1.0
)

on a side note, the speed seems lower - like a deeper voice when using RealtimeTTS imports, I could not find a setting from the engine or the player to make it higher pitch or faster

abd thank you so much for this project!

**Comment by KoljaB** [comment]
Oh, I wasn't aware that OpenAI supports that. Will integrate soon, thanks for pointing me to this.
